src/requests.ts

```diff
@@ -175,6 +178,37 @@ export type ApiRequestType<
   | IPostApiRequestType<P, KH, Q, R>
   | IPutApiRequestType<P, KH, Q, R>;
 
+/**
+ * The type of the Params of an ApiRequestType
+ */
+export type TypeofApiParams<T> = T extends ApiRequestType<
+  infer P,
+  infer KH,
+  infer Q,
+  infer R
+>
+  ? P
+  : never;
+
+/**
+ * The type of the Response of an ApiRequestType
+ */
+export type TypeofApiResponse<T> = T extends ApiRequestType<
+  infer P,
+  infer KH,
+  infer Q,
+  infer R
+>
+  ? R
+  : never;
+
+/**
+ * The type of the Response of an ApiRequestType
```

Comment:
not clear, is the same comment as above

Reply:
https://github.com/teamdigitale/italia-ts-commons/commit/4a8ee01e0fbf144e75be8c4e25530b733881ba99